### PR TITLE
Device instances

### PR DESCRIPTION
**Pull request by marcelveldt**
Split up device with multiple instances into virtual devices for each instance.

### CODE REVIEW

**Comment by marcelveldt** [comment]
Closes #63

File: custom_components/zwave_mqtt/entity.py [hunk 1]
```diff
@@ -153,12 +153,23 @@ async def async_added_to_hass(self):
...
+        if instance > 1:
+            return {
+                "identifiers": {(DOMAIN, node.node_id, instance)},
+                "name": f"{node.node_manufacturer_name} {node.node_product_name} - Instance {instance}",
+                "manufacturer": node.node_manufacturer_name,
+                "model": node.node_product_name,
+                "via_device": (DOMAIN, node.node_id),
+            }
+        else:
+            return {
```
Comment (MartinHjelmare) [inline custom_components/zwave_mqtt/entity.py]: We can create this dict first and if there are more than one instance modify the needed items in the dict before returning it.

Comment (marcelveldt) [reply custom_components/zwave_mqtt/entity.py]: Better indeed, I've just adjusted that.

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

**Pull request merged**